README.md

```diff
@@ -22,6 +23,122 @@ git add -A
 
 This section provides instructions on how you can enable CI for your repository.
 
+### Github Actions
+
+These are instructions when using [Github
+Actions](https://docs.github.com/en/free-pro-team@latest/actions).
+
+The following configuration shows how to:
+
+* Upload nuget packages to a NuGet feed for tags, and commits on `main` branch.
+
+  This requires adding a secret named `NUGET_API_KEY` to your
+  repository on GitHub.
+
+  Add a `.github/workflows/ci.yml` to your repository:
+
+  ``` yml
+  name: Build, test and publish NuGet packages
+
+  on:
+  push:
+      branches: [ main ]
+      tags:
+      - '*'
+
+  jobs:
+  build:
+      name: Build, test and publish NuGet package
+
+      runs-on: ubuntu-latest
+
+      steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+          dotnet-version: '5.0.x'
+      - name: Restore dependencies
+      run: dotnet restore
+      - name: Build
+      run: dotnet build --no-restore
```

Comment:
I'd like this to be similar to the `.travis.yml`. Can you remove the separate restore step?
Also the order of build and test should be the same. Can you update the one in the README for travis to be build first, and then test, like what you have here.